javascriptAddTest/test.js

```diff
@@ -0,0 +1,22 @@
+const add = require ("./app.js");
+
+const test = (value1, value2) => {
+
+  if (add(value1, value2) === (value1 + value2)) {
```

Comment:
Instead of using value1 + value2 on the right, use a third variable so it can be compared with the added code.